source/dub/packagemanager.d

```diff
@@ -188,9 +188,11 @@ class PackageManager {
 	/// ditto
 	Package getPackage(string name, Version ver, NativePath path)
 	{
-		auto ret = getPackage(name, path);
-		if (!ret || ret.version_ != ver) return null;
-		return ret;
+		foreach( p; getPackageIterator(name) )
+			if (p.version_ == ver)
+				if (p.path.startsWith(path))
```

Comment:
a little bit picking here, but I think writing this with `&&` instead of a nested `if` would be better style